API_PATTERN_GUIDELINES.md

```diff
@@ -24,6 +24,7 @@ Some of the below rules are inspired by painpoints we've encountered in this pro
 - [Code Style](#code-style)
   - [Default Props](#default-props)
   - [Class Function Binding](#class-function-binding)
+  - [Arrow and Bound Functions in Render](#arrow-and-bound-functions-in-render)
```

Comment:
looks like some of these changes are also being made in https://github.com/Workday/canvas-kit/pull/276 ?